stable/lamp/README.md

```diff
@@ -129,8 +129,7 @@ $ im-done
 | Parameter | Description | Default |
 | - | - | - |
 | `init.manually.enabled` | Enables container for manual initialization | false |
-| `init.manually.repository` | Containers repository | lead4good/init-wp |
-| `init.manually.tag` | Repository tag | latest |
+| `init.manually.image` | Containers image | lead4good/init-wp:latest |
```

Comment:
this would be a breaking change and would require semver bumped a full version (or at least a minor since this is a not even version 1 ).